js/app/application.js

```diff
@@ -43,6 +43,19 @@ const KnowledgeSearchIface = '\
   </interface> \
 </node>';
 
+const GrandCentralContentIface = '\
+<node> \
+  <interface name="com.endlessm.GrandCentralContent"> \
+    <method name="ContentQuery"> \
+      <arg type="s" name="Query" direction="in" /> \
+      <arg type="s" name="Result" direction="out" /> \
+    </method> \
+    <method name="QueryNotifications"> \
+      <arg name="notifications" direction="out" type="aa{ss}" /> \
+    </method> \
```

Comment:
I don't think this API should have anything related to notifications. Notifications already have their own DBus API and grand central should tap into that.